Assets/Plugins/Source/Editor/Internal/UnityPackageCreationTool.cs

```diff
@@ -22,6 +22,8 @@
 
 using UnityEngine;
 using UnityEditor;
+using System.IO;
+using Playeveryware.Editor;
```

Comment:
`Playeveryware.Editor` should be fixed to be `PlayEveryWare.EpicOnlineServices.Editor`. Despite the fact that this PR doesn't _introduce_ the creation of this `namespace`, when a namespace usage like this is discovered we should correct it. As it stands, when you fix the conflicts with `development`, it's possible that one of my PRs will have already fixed this.